features/book-index-page.feature

```diff
@@ -16,7 +16,7 @@ Feature: Book index store page
     Then the books are re-sorted based on the amount of times they are purchased
 
   Scenario: Book searching
-    Given there are 100 books in the database
+    Given there are 10 books in the database
```

Comment:
I had an issue where something wasn't showing up and I thought it might have been because I had my test sorted ascending and the thing I was looking for was on another page, so I made fewer books in order to test that hypothesis. Turns out I was just doing it wrong. :)